src/main/java/duke/Command.java

```diff
@@ -0,0 +1,211 @@
+package duke;
+
+import duke.exception.DetailsUnknownException;
+import duke.exception.IncompleteMessageException;
+import duke.exception.InvalidTaskNumberException;
+
+/**
+ * Command is responsible for handling the different command
+ * enter by the user.
+ */
+public class Command {
```

Comment:
Command should be an abstract class with only two methods: isExit() and execute(). Should create classes like FindCommand or addComand that extend Command